### PR TITLE
Block reward address wrapper added

### PR DESCRIPTION
**Pull request by H-Pavel**
# Checklist:

- [ ] Code follows the style guidelines
- [ ] Documentation updated
- [ ] Corner cases evaluated and handled
- [ ] index.js exports updated
- [ ] Automatic tests added/updated
- [ ] Automatic tests pass
- [ ] Sanity check passed
- [ ] Self-review passed
- [ ] Version updated

### CODE REVIEW

File: test/wrappers/validator_proxy_wrapper.js [hunk 1]
```diff
@@ -125,4 +125,28 @@ describe('ValidatorProxy Wrapper', () => {
       expect(setBaseRewardSendStub).to.be.calledOnceWith({from: validatorProxyOwner});
     });
   });
+
+  describe('getBlockRewardsContractAddress', () => {
+    let blockRewardsStub;
+    let blockRewardsCallStub;
+    const validatorProxyOwner = '0x1234ABCD';
```
Comment (krzysztof-jelski) [inline test/wrappers/validator_proxy_wrapper.js]:
```suggestion
    const blockRewardsAddress = '0x1234ABCD';
```

Comment (H-Pavel) [reply test/wrappers/validator_proxy_wrapper.js]: Fixed
